test/unit/parameterized-url.test.js

```diff
@@ -1,4 +1,4 @@
-const { BaseService } = require('../../dist/lib/base-service');
+const { constructServiceURL } = require('../../dist/lib/helper');
```

Comment:
Question: will moving this function to helpers.ts make it more difficult to import it in the generated SDK code?
@dpopp07 any idea?

Reply:
@padamstx Good question - no, it won't. All of the helper methods are exported at the top level of the core so the import will be simple:

```node
const { constructServiceUrl, <any-other-imports> } = require('ibm-cloud-sdk-core');
```

Reply:
ok, sounds good, but I assume this means that when we generate the service code for a service, we'll need to be careful about correctly importing `constructServiceUrl` from the core when we will be generating a service-specific `constructServiceUrl()` function that will reference the function in the core.

Reply:
Oh, because of name conflicts? That could be a legitimate issue, though it should be easy to get around in a compatible way

Reply:
I wasn't thinking about name conflicts actually.  But in the node generator, when we process an API definition, we'd generate the constructServiceUrl() function in the service .ts file ONLY if we detect server variables in one of the "servers" entries.
So only in that case would we want to import the "constructServiceUrl" function from the core, since that's the only scenario where we'd use it.

Reply:
Oh, I see. That makes sense